src/core_plugins/kibana/mappings.json

```diff
@@ -167,5 +167,15 @@
         "type": "keyword"
       }
     }
+  },
+  "kql-telemetry": {
```

Comment:
Are you looking for a global count of all opt ins to `kql-telemetry`? If that's the case, you'll want to add something similar to the kibana plugin's uiExports copied from [here](https://github.com/elastic/kibana/blob/spaces-phase-1/x-pack/plugins/xpack_main/index.js#L85):
```
      savedObjectSchemas: {
        kql-telemetry: {
          isNamespaceAgnostic: true,
        },
      },
```

otherwise, we'll be inserting a document per "Space" once that merges and when a user has opted into it.

Reply:
I assume we want it to be global... but what does a namespace represent exactly? Is it for workspaces?

Reply:
Yeah, namespaces are to support Spaces (at this moment). Whenever this is a Space, we use the namespace to differentiate between the saved objects.